app/import_validators_test.go

```diff
@@ -692,6 +692,12 @@ func TestImportValidateUserTeamsImportData(t *testing.T) {
 	if err := validateUserTeamsImportData(&data); err != nil {
 		t.Fatal("Should have succeeded with valid roles.")
 	}
+
```

Comment:
Would be nice to test the error case too for "not valid json", or for "valid json with invalid Theme data".

Reply:
I haven't seen any theme validation yet. Could you please point me the direction where I could read about standards/restrictions? Or did you mean to just check fields set to contain all keys currently present in the test string?

Reply:
In case this paragraph(https://docs.mattermost.com/help/settings/theme-colors.html#custom-themes) contains all fields set describing an abstract theme, I could add it to the model as JSON object with according marshaling and validation function. The only difference it has - the way to store and to process it as string but not as JSON object

Reply:
Look please to last 2 commits and let me know if we need to update the model to make theme string validation more strict and precise.

Reply:
Nope, that is enough for now. The scope of this PR is not add Theme structure validation. Would be nice to have something like that as part of another PR, but at least for now, is not needed.